packages/pigeon/run_tests.sh

```diff
@@ -279,16 +283,24 @@ run_ios_unittests() {
     --dart_out /dev/null \
     --objc_header_out platform_tests/ios_unit_tests/ios/Runner/messages.h \
     --objc_source_out platform_tests/ios_unit_tests/ios/Runner/messages.m
+  clang-format -i platform_tests/ios_unit_tests/ios/Runner/messages.h
+  clang-format -i platform_tests/ios_unit_tests/ios/Runner/messages.m
   pub run pigeon \
     --no-dart_null_safety \
     --input pigeons/async_handlers.dart \
     --dart_out /dev/null \
     --objc_header_out platform_tests/ios_unit_tests/ios/Runner/async_handlers.h \
     --objc_source_out platform_tests/ios_unit_tests/ios/Runner/async_handlers.m
-  clang-format -i platform_tests/ios_unit_tests/ios/Runner/messages.h
-  clang-format -i platform_tests/ios_unit_tests/ios/Runner/messages.m
   clang-format -i platform_tests/ios_unit_tests/ios/Runner/async_handlers.h
   clang-format -i platform_tests/ios_unit_tests/ios/Runner/async_handlers.m
+  pub run pigeon \
+    --no-dart_null_safety \
+    --input pigeons/all_datatypes.dart \
+    --dart_out /dev/null \
+    --objc_header_out platform_tests/ios_unit_tests/ios/Runner/all_datatypes.h \
+    --objc_source_out platform_tests/ios_unit_tests/ios/Runner/all_datatypes.m    
+  clang-format -i platform_tests/ios_unit_tests/ios/Runner/all_datatypes.h
+  clang-format -i platform_tests/ios_unit_tests/ios/Runner/all_datatypes.m
   pushd $PWD
   cd platform_tests/ios_unit_tests
```

Comment:
Not for this PR, but it would be useful to look into whether the native unit tests could be using `flutter_plugin_tools`, which has pretty similar logic for running native unit tests for plugins.